compiler/noirc_frontend/src/hir/comptime/interpreter/builtin.rs

```diff
@@ -444,6 +483,20 @@ fn type_eq(mut arguments: Vec<(Value, Location)>, location: Location) -> IResult
     Ok(Value::Bool(value1 == value2))
 }
 
+// fn is_field(self) -> bool
+fn type_is_field(mut arguments: Vec<(Value, Location)>, location: Location) -> IResult<Value> {
+    check_argument_count(1, &arguments, location)?;
+
+    let value = arguments.pop().unwrap().0;
+    let Value::Type(typ) = value else {
+        panic!("type_is_field shold have been called with a type");
+    };
+
+    let is_field = matches!(typ, Type::FieldElement);
+
+    Ok(Value::Bool(is_field))
```

Comment:
```suggestion
    Ok(Value::Bool(matches!(typ, Type::FieldElement)))
```
I like to keep line count low